lib/govuk_index/presenters/details_presenter.rb

```diff
@@ -1,48 +1,30 @@
 module GovukIndex
   class DetailsPresenter
```

Comment:
I think we should consider renaming this class, because `details` isn't something we're presenting, it's something we're consuming.

I think the job of this class is to make the incoming payload easier to pull apart.

In general, there's no reason why we couldn't have fields in our elasticsearch documents that are derived partly from the details hash and partly from other bits of the content item.

`indexable_content` in particular is one of the most complicated fields, and it's also really important. Maybe that should live in the main presenter, and delegate to this one for `indexable_content_parts`?

Reply:
+1

@brenetic mentioned earlier that it seemed easier to structure the presenters around the payload structure when we originally started the mapping, but it might not make sense anymore when we have to map several very different formats.

Reply:
I think we are still some way off on naming, however I think the current split is working pretty well. I will leave off renaming this class right now, with the plan to look at the name of this class next time we add any functionality to it (I had originally planned to put the specialist formats fields in this class, but changed my mind in the end.